src/Features/Core/Portable/CodeRefactorings/CodeRefactoringService.cs

```diff
@@ -306,5 +339,26 @@ private ImmutableArray<CodeRefactoringProvider> CreateRefactorings(string langua
                 return builder.ToImmutableAndFree();
             }
         }
+
+        private sealed class CodeRefactoringServiceCallback
```

Comment:
would it be cleaner to just pass the pasted text span over when we call into OOP?  it seems simpler than needing the callback and for OOP to need to call back in just for this one piece of data.

Reply:
Done